tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "sourceMap": true,
+    "target": "es5",
+    "outDir": "lib",
+    "declaration": true
+  },
+  "files": [
+    "src/mediator.ts",
```

Comment:
Agree, I guess leaving only `files` in `package.json` in order to publish a slimmer module would be the best way to go, adds another point of change when adding a new file.

Going to add strict null checks. I had removed implicit `any`s as well but I couldn't get past an external dependency that doesn't have definitions.